### PR TITLE
Feature/39320

### PR DESCRIPTION
**Pull request by Aekal**
Internal: 39320

### CODE REVIEW

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss [hunk 1]
```diff
@@ -154,14 +184,20 @@ $catalog-grid-item__action-icon-size        : 24px !default;
...
+        &:hover,
+        &:focus {
+            .icon {
+                fill: $green;
+            }
+        }
         .icon {
             width: $catalog-grid-item__action-icon-size;
             height: $catalog-grid-item__action-icon-size;
+            transition: $transition-base;
```
Comment (szafran89) [inline components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: move to component variable

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss [hunk 2]
```diff
@@ -154,14 +184,20 @@ $catalog-grid-item__action-icon-size        : 24px !default;
...
         width: $catalog-grid-item__action-size;
-        margin-left: $catalog-grid-item__base-spacing;
         padding: 0;
         background: transparent;
         border: none;
         cursor: pointer;
+        &:hover,
+        &:focus {
+            .icon {
+                fill: $green;
```
Comment (szafran89) [inline components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: move to component variable

File: components/03-modules/filters/_filters.scss [hunk 3]
```diff
@@ -28,19 +31,21 @@ $filters__list-margin                : $spacer--medium 0 0 0 !default;
...
         padding: $filters__switcher-padding;
-        border: $filters__switcher-border;
+        border: none;
+        border-top: $filters__switcher-border;
         border-radius: $filters__switcher-border-radius;
         background-color: $filters__switcher-background-color;
         color: $filters__switcher-color;
         font-size: $filters__switcher-font-size;
-        font-weight: $filters__switcher-font-weight;
+        font-family: $font-family-base;
```
Comment (szafran89) [inline components/03-modules/filters/_filters.scss]: move to variable

File: components/03-modules/filters/_filters.scss [hunk 4]
```diff
@@ -54,13 +59,31 @@ $filters__list-margin                : $spacer--medium 0 0 0 !default;
         }
     }
 
+    &__switcher-icon {
+        margin-left: $filters__switcher-icon-margin;
+        padding: $filters__switcher-icon-padding;
+        transition: $transition-base;
```
Comment (szafran89) [inline components/03-modules/filters/_filters.scss]: move to variable

File: components/03-modules/filters/_filters.scss [hunk 5]
```diff
@@ -54,13 +59,31 @@ $filters__list-margin                : $spacer--medium 0 0 0 !default;
...
         &--mobile-hidden {
             display: none;
             @include mq($screen-m) {
                 display: block;
             }
         }
     }
+
+    &__active {
+        margin: 0 0 $spacer--medium 0;
```
Comment (szafran89) [inline components/03-modules/filters/_filters.scss]: move to component variable

File: components/03-modules/range/_range.scss [hunk 6]
```diff
@@ -42,10 +43,15 @@ $range__suffix-padding        : 0 $spacer !default;
...
     }
+    &__button {
+        margin: $range__margin;
+        padding: $range__button-padding;
+    }
 }
 
 .noUi-connect {
     background-color: $range__track-bg-color--active;
+    border-radius: 0;
```
Comment (szafran89) [inline components/03-modules/range/_range.scss]: move to component variable

File: components/03-modules/toolbar/_toolbar.scss [hunk 7]
```diff
@@ -91,12 +94,15 @@ $toolbar__sorter-padding           : 8px 0 !default;
...
-    &__mode-link {
+    &__mode-button {
         height: $toolbar__mode-icon-area-size;
         width: $toolbar__mode-icon-area-size;
         display: flex;
         justify-content: center;
         align-items: center;
+        background: none;
+        border: none;
+        cursor: pointer;
```
Comment (szafran89) [inline components/03-modules/toolbar/_toolbar.scss]: move above to variables

File: components/03-modules/toolbar/_toolbar.scss [hunk 8]
```diff
@@ -138,6 +153,21 @@ $toolbar__sorter-padding           : 8px 0 !default;
...
+            width: 200px;
+        }
+    }
+
+    &__limiter-select {
+        @include mq($screen_m) {
+            width: 72px;
+        }
+    }
+
```
Comment (szafran89) [inline components/03-modules/toolbar/_toolbar.scss]: move above to variables

File: components/04-views/catalog/grid/_grid.scss [hunk 9]
```diff
@@ -1,11 +1,14 @@
...
+$catalog-grid__padding: $spacer--medium 0 $spacer--large 0;
+$catalog-grid__border : 1px solid $gray-light;
 
 .catalog-grid {
     display: grid;
     grid-template-columns: 1fr 1fr;
     grid-gap: $catalog-grid__gap;
     margin: 0;
-    padding: 0;
+    padding: $catalog-grid__padding;
```
Comment (szafran89) [inline components/04-views/catalog/grid/_grid.scss]: move to variable

File: components/04-views/catalog/grid/_grid.scss [hunk 10]
```diff
@@ -1,11 +1,14 @@
...
+    border-bottom: $catalog-grid__border;
```
Comment (szafran89) [inline components/04-views/catalog/grid/_grid.scss]: move to variable

File: components/04-views/catalog/grid/grid.hbs [hunk 11]
```diff
@@ -2,20 +2,26 @@
...
-
-    {{ render '@pager' }}
+    <div class="catalog-list">
+        <div class="catalog-list__sidebar">
+            {{ render '@filters' }}
+        </div>
+        <div class="catalog-list__products">
+            {{ render '@toolbar' }}
+            <ul class="catalog-grid">
+                {{ render '@catalog-grid-item' product merge=true }}
```
Comment (szafran89) [inline components/04-views/catalog/grid/grid.hbs]: Do we need `merge=true`?

File: components/04-views/catalog/grid/grid.hbs [hunk 12]
```diff
@@ -2,20 +2,26 @@
...
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+            </ul>
+            {{ render '@pager' }}
```
Comment (szafran89) [inline components/04-views/catalog/grid/grid.hbs]: can we move pager outside `catalog-list__products`?

File: components/04-views/catalog/list/_list.scss [hunk 13]
```diff
@@ -33,7 +35,9 @@ $catalog-list__products-padding       : 0 0 $spacer--large 0 !default;
         &:first-child {
             border-top: $catalog-list__border;
         }
-
+        &:first-of-type {
+            padding-top: $spacer--medium;
```
Comment (szafran89) [inline components/04-views/catalog/list/_list.scss]: move to variable

File: components/02-elements/button/button.hbs [hunk 14]
```diff
@@ -1,5 +1,5 @@
 <{{{ tag }}} class="button {{ class }}" {{{ attributes }}}>
-    {{ text }}
+    {{{ text }}}
```
[the remark this answers is not in the document]
Comment (Aekal) [reply components/02-elements/button/button.hbs]: I couldn't pass a `span` to `button`, so I had to change to triple brackets. We have the same solution e.g. in `checkbox.hbs` or `cookie-message.hbs`

File: components/03-modules/active-filters/active-filters.hbs [hunk 15]
```diff
@@ -1,12 +1,13 @@
 <ul class="active-filters list {{class}}">
     {{# each filters }}
         <li class="active-filters__item list__item {{class}}">
-            <button type="button"
-                    class="active-filters__remove {{class}}"
-                    aria-label="{{ariaLabel}}"
+            <a
+                class="active-filters__remove {{class}}"
+                aria-label="{{ariaLabel}}"
+                title="{{ariaLabel}}"
```
[the remark this answers is not in the document]
Comment (Aekal) [reply components/03-modules/active-filters/active-filters.hbs]: Added

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss [hunk 16]
```diff
@@ -5,8 +5,11 @@ $catalog-grid-item__font-size               : $font-size-small !default;
 $catalog-grid-item__font-size\@large        : $font-size-base !default;
 $catalog-grid-item__line-height             : 1.5 !default;
 
+$catalog-grid-item__badge-top-position      : 16px !default;
```
[the remark this answers is not in the document]
Comment (Aekal) [reply components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: Fixed

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss (hunk 1 above)
Comment (Aekal) [reply components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: Done

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss (hunk 2 above)
Comment (Aekal) [reply components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: Done

File: components/03-modules/filters/_filters.scss (hunk 3 above)
Comment (Aekal) [reply components/03-modules/filters/_filters.scss]: Done

File: components/03-modules/filters/_filters.scss (hunk 4 above)
Comment (Aekal) [reply components/03-modules/filters/_filters.scss]: Done

File: components/03-modules/filters/_filters.scss (hunk 5 above)
Comment (Aekal) [reply components/03-modules/filters/_filters.scss]: Done

File: components/03-modules/range/_range.scss (hunk 6 above)
Comment (Aekal) [reply components/03-modules/range/_range.scss]: Done

File: components/03-modules/toolbar/_toolbar.scss (hunk 8 above)
Comment (Aekal) [reply components/03-modules/toolbar/_toolbar.scss]: Done

File: components/04-views/catalog/grid/_grid.scss (hunk 9 above)
Comment (Aekal) [reply components/04-views/catalog/grid/_grid.scss]: Done

File: components/04-views/catalog/grid/_grid.scss (hunk 10 above)
Comment (Aekal) [reply components/04-views/catalog/grid/_grid.scss]: Done

File: components/04-views/catalog/grid/grid.hbs (hunk 11 above)
Comment (Aekal) [reply components/04-views/catalog/grid/grid.hbs]: Done, I removed configs for list and grid view

File: components/04-views/catalog/list/_list.scss (hunk 13 above)
Comment (Aekal) [reply components/04-views/catalog/list/_list.scss]: Done

File: components/04-views/catalog/grid/grid.hbs (hunk 12 above)
Comment (Aekal) [reply components/04-views/catalog/grid/grid.hbs]: It's better to leave it here to fit pager in grid

File: components/02-elements/button/button.hbs (hunk 14 above)
[the remark this answers is not in the document]
Comment (szafran89) [reply components/02-elements/button/button.hbs]: Yes it's ok here, there are unnecessary triple braces for attributes and tag but we will change it later.

File: components/03-modules/active-filters/active-filters.hbs [hunk 17]
```diff
@@ -1,12 +1,14 @@
...
+                href="#"
+                class="active-filters__remove {{class}}"
+                aria-label="{{ariaLabel}}"
+                title="{{ariaLabel}}"
             >
                 {{ render '@icon' icon }}
-            </button>
+            </a>
             <span class="active-filters__label">
                 {{label}}&nbsp;:&nbsp;
```
Comment (szafran89) [inline components/03-modules/active-filters/active-filters.hbs]: spaces `{{ label }}`

Comment (szafran89) [reply components/03-modules/active-filters/active-filters.hbs]: `&nbsp;:&nbsp;` it looks a little weird, can we replace it?

File: components/03-modules/active-filters/active-filters.hbs [hunk 18]
```diff
@@ -15,8 +17,10 @@
             </span>
         </li>
     {{/each}}
-    <a href="{{clearHref}}"
-       class="active-filters__clear-all {{clearClass}}"
+    <a
+        href="{{clearHref}}"
```
Comment (szafran89) [inline components/03-modules/active-filters/active-filters.hbs]: add spaces `{{ clearHref }}`

File: components/03-modules/active-filters/active-filters.hbs [hunk 19]
```diff
@@ -15,8 +17,10 @@
...
+        class="active-filters__clear-all {{clearClass}}"
```
Comment (szafran89) [inline components/03-modules/active-filters/active-filters.hbs]: `{{ clearClass }}` spaces

File: components/03-modules/active-filters/active-filters.hbs [hunk 20]
```diff
@@ -15,8 +17,10 @@
...
+        title="Clear all filters"
```
Comment (szafran89) [inline components/03-modules/active-filters/active-filters.hbs]: move to config

File: components/03-modules/active-filters/active-filters.hbs [hunk 21]
```diff
@@ -15,8 +17,10 @@
...
     >
         {{clearText}}
```
Comment (szafran89) [inline components/03-modules/active-filters/active-filters.hbs]: add spaces `{{ clearText }}`

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss [hunk 22]
```diff
@@ -88,13 +98,16 @@ $catalog-grid-item__action-icon-size        : 24px !default;
...
 
+    &__rating {
+        margin: $catalog-grid-item__rating-margin;
+    }
+
     &__price {
-        padding-left: $catalog-grid-item__base-spacing;
         font-size: $catalog-grid-item__font-size;
         line-height: $catalog-grid-item__line-height;
         font-weight: $font-weight-bold;
```
Comment (szafran89) [inline components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: move `font-weight` to var

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss [hunk 23]
```diff
@@ -142,10 +153,31 @@ $catalog-grid-item__action-icon-size        : 24px !default;
         width: 100%;
     }
 
+    &__primary-form {
+        margin: 0;
```
Comment (szafran89) [inline components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: move to var

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss [hunk 24]
```diff
@@ -154,14 +186,20 @@ $catalog-grid-item__action-icon-size        : 24px !default;
         justify-content: center;
         height: $catalog-grid-item__action-size;
         width: $catalog-grid-item__action-size;
-        margin-left: $catalog-grid-item__base-spacing;
         padding: 0;
         background: transparent;
         border: none;
         cursor: pointer;
```
Comment (szafran89) [inline components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: move above to variables

File: components/03-modules/catalog-grid-item/catalog-grid-item.hbs [hunk 25]
```diff
@@ -9,11 +9,7 @@
...
     </a>
 
-    <section class="
-                 catalog-grid-item__flex-wrapper
-                 catalog-grid-item__main
-             "
-    >
+    <section class="catalog-grid-item__main">
         <h2 class="catalog-grid-item__name">
             <a href="#"
```
Comment (szafran89) [inline components/03-modules/catalog-grid-item/catalog-grid-item.hbs]: attr to new line

File: components/03-modules/catalog-grid-item/catalog-grid-item.hbs [hunk 26]
```diff
@@ -27,22 +23,18 @@
...
-                   class="catalog-grid-item__link"
-                >
-                    {{ name }}
-                </a>
-            </div>
-
-            <div class="catalog-grid-item__price">
-                {{ render (component price.name) price.context }}
-            </div>
+        <div class="
```
Comment (szafran89) [inline components/03-modules/catalog-grid-item/catalog-grid-item.hbs]: class to new line

File: components/03-modules/range/_range.scss [hunk 27]
```diff
@@ -42,12 +44,15 @@ $range__suffix-padding        : 0 $spacer !default;
         flex-grow: 1;
         margin-bottom: 0;
```
Comment (szafran89) [inline components/03-modules/range/_range.scss]: move to var

File: components/03-modules/range/_range.scss [hunk 28]
```diff
@@ -42,12 +44,15 @@ $range__suffix-padding        : 0 $spacer !default;
...
     }
+    &__button {
+        margin: $range__margin;
```
Comment (szafran89) [inline components/03-modules/range/_range.scss]: change to `$range__button-margin`

File: components/03-modules/toolbar/_toolbar.scss [hunk 29]
```diff
@@ -131,10 +146,18 @@ $toolbar__sorter-padding           : 8px 0 !default;
...
     }
 
-    &__limiter-wrapper {
-        margin-bottom: 0;
-        @include mq($screen-m) {
-            margin-right: $toolbar__offset;
+    &__sorter-select,
+    &__limiter-select {
+        height: 32px;
+        font-family: $font-family-base;
```
Comment (szafran89) [inline components/03-modules/toolbar/_toolbar.scss]: move to var

File: components/03-modules/toolbar/_toolbar.scss [hunk 30]
```diff
@@ -131,10 +146,18 @@ $toolbar__sorter-padding           : 8px 0 !default;
         line-height: $toolbar__select-field-item-height;
     }
 
-    &__limiter-wrapper {
-        margin-bottom: 0;
-        @include mq($screen-m) {
-            margin-right: $toolbar__offset;
+    &__sorter-select,
+    &__limiter-select {
+        height: 32px;
```
Comment (szafran89) [inline components/03-modules/toolbar/_toolbar.scss]: move to var

File: components/03-modules/toolbar/_toolbar.scss [hunk 31]
```diff
@@ -131,10 +146,18 @@ $toolbar__sorter-padding           : 8px 0 !default;
...
+        font-family: $font-family-base;
+        @include mq($screen_m) {
+            width: 200px;
+        }
+    }
+
+    &__limiter-select {
+        @include mq($screen_m) {
+            width: 72px;
```
Comment (szafran89) [inline components/03-modules/toolbar/_toolbar.scss]: move to var

File: components/03-modules/toolbar/_toolbar.scss [hunk 32]
```diff
@@ -131,10 +146,18 @@ $toolbar__sorter-padding           : 8px 0 !default;
...
-    &__limiter-wrapper {
-        margin-bottom: 0;
-        @include mq($screen-m) {
-            margin-right: $toolbar__offset;
+    &__sorter-select,
+    &__limiter-select {
+        height: 32px;
+        font-family: $font-family-base;
+        @include mq($screen_m) {
+            width: 200px;
```
Comment (szafran89) [inline components/03-modules/toolbar/_toolbar.scss]: move to var

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss [hunk 33]
```diff
@@ -88,13 +98,16 @@ $catalog-grid-item__action-icon-size        : 24px !default;
...
+    }
+
     &__price {
-        padding-left: $catalog-grid-item__base-spacing;
         font-size: $catalog-grid-item__font-size;
         line-height: $catalog-grid-item__line-height;
         font-weight: $font-weight-bold;
-        text-align: right;
         white-space: nowrap;
+        padding-bottom: $spacer;
```
[the remark this answers is not in the document]
Comment (Aekal) [reply components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: Done

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss [hunk 34]
```diff
@@ -15,25 +18,37 @@ $catalog-grid-item__content-padding\@medium : $spacer--medium !default;
...
+                transform: translateY(-100%);
+            }
         }
     }
 
     &__badge {
         position: absolute;
         top: $catalog-grid-item__badge-top-position;
         left: $catalog-grid-item__badge-left-position;
+        border-radius: 0;
```
[the remark this answers is not in the document]
Comment (Aekal) [reply components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: Done

File: components/03-modules/active-filters/active-filters.hbs (hunk 17 above)
Comment (Aekal) [reply components/03-modules/active-filters/active-filters.hbs]: I left one &nbsp

File: components/03-modules/active-filters/active-filters.hbs (hunk 18 above)
Comment (Aekal) [reply components/03-modules/active-filters/active-filters.hbs]: Done

File: components/03-modules/active-filters/active-filters.hbs (hunk 19 above)
Comment (Aekal) [reply components/03-modules/active-filters/active-filters.hbs]: Done

File: components/03-modules/active-filters/active-filters.hbs (hunk 20 above)
Comment (Aekal) [reply components/03-modules/active-filters/active-filters.hbs]: Done

File: components/03-modules/active-filters/active-filters.hbs (hunk 21 above)
Comment (Aekal) [reply components/03-modules/active-filters/active-filters.hbs]: Done

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss (hunk 22 above)
Comment (Aekal) [reply components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: Done

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss (hunk 23 above)
Comment (Aekal) [reply components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: Done

File: components/03-modules/catalog-grid-item/_catalog-grid-item.scss (hunk 24 above)
Comment (Aekal) [reply components/03-modules/catalog-grid-item/_catalog-grid-item.scss]: Done

File: components/03-modules/catalog-grid-item/catalog-grid-item.hbs (hunk 25 above)
Comment (Aekal) [reply components/03-modules/catalog-grid-item/catalog-grid-item.hbs]: Done

File: components/03-modules/catalog-grid-item/catalog-grid-item.hbs (hunk 26 above)
Comment (Aekal) [reply components/03-modules/catalog-grid-item/catalog-grid-item.hbs]: Done

File: components/03-modules/range/_range.scss (hunk 27 above)
Comment (Aekal) [reply components/03-modules/range/_range.scss]: Done

File: components/03-modules/range/_range.scss (hunk 28 above)
Comment (Aekal) [reply components/03-modules/range/_range.scss]: Done

File: components/03-modules/toolbar/_toolbar.scss (hunk 29 above)
Comment (Aekal) [reply components/03-modules/toolbar/_toolbar.scss]: Done

File: components/03-modules/toolbar/_toolbar.scss (hunk 30 above)
Comment (Aekal) [reply components/03-modules/toolbar/_toolbar.scss]: Done

File: components/03-modules/toolbar/_toolbar.scss (hunk 31 above)
Comment (Aekal) [reply components/03-modules/toolbar/_toolbar.scss]: Done

File: components/03-modules/toolbar/_toolbar.scss (hunk 32 above)
Comment (Aekal) [reply components/03-modules/toolbar/_toolbar.scss]: Done

**Pull request merged**